src/storage/local_storage/local_table.cpp

```diff
@@ -89,142 +55,53 @@ void LocalVectorCollection::prepareAppend() {
 }
 
 LocalNodeGroup::LocalNodeGroup(std::vector<LogicalType*> dataTypes, MemoryManager* mm) {
-    columns.resize(dataTypes.size());
+    chunks.resize(dataTypes.size());
```

Comment:
Better to use reserve, and then call append in the for loop.
resize requires that the elements inside chunks have a valid copy constructor.